metabase/metabase
# Remove animation from notebook editor under reduced motion

Solves #18172.

## How to test

1. Ask a Question
2. Simple Question
3. Sample Dataset
4. People
5. On the top right, click on icon to "Show Editor"

If you have `prefers-reduced-motion` set to `true`, the notebook editor should pop in and out with no slide animation.

In macOS, you can set `prefers-reduced-motion` by going to the **Settings app › Accessibility › Display › Reduce Motion**.

[Other ways to set `prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion#user_preferences).

![image](https://user-images.githubusercontent.com/380816/135523416-4f90d1b9-e0d9-48c5-ba35-c47103e1662f.png)
